Find a functionally equivalent code:com.android.tools.layoutlib.create.AsmAnalyzer.findDeps:COMMENT
Method Modifier: 
Comment:/**
 * Finds all dependencies for all classes in keepClasses which are also
 * listed in zipClasses. Returns a map of all the dependencies found.
 */

@@ -5,10 +5,11 @@
     TreeMap<String, ClassReader> temp = new TreeMap<String, ClassReader>();
     DependencyVisitor visitor = getVisitor(zipClasses, inOutKeepClasses, new_keep, deps, new_deps);
     for (ClassReader cr : inOutKeepClasses.values()) {
+        visitor.setClassName(cr.getClassName());
         cr.accept(visitor, 0);
     }
     while (new_deps.size() > 0 || new_keep.size() > 0) {
@@ -17,22 +18,24 @@
         new_keep.clear();
         mLog.debug("Found %1$d to keep, %2$d dependencies.", inOutKeepClasses.size(), deps.size());
         for (ClassReader cr : temp.values()) {
+            visitor.setClassName(cr.getClassName());
             cr.accept(visitor, 0);
         }
     }

